Dockerfile

```diff
@@ -44,12 +39,12 @@ FROM rust:1.54.0-alpine as base-optimizer
 RUN apk update && \
   apk add --no-cache musl-dev
 
+# Install platform-specific binaryen
+RUN apk add binaryen
+
 # Setup Rust with Wasm support
 RUN rustup target add wasm32-unknown-unknown
 
-# Add wasm-opt
-COPY --from=builder /usr/local/bin/wasm-opt /usr/local/bin
```

Comment:
If we stick with the copy command we don't need to ship all unused `binaryen` tools to the user (https://pkgs.alpinelinux.org/contents?file=&path=&name=binaryen&branch=v3.14&repo=community&arch=x86)

Reply:
Yes, I tried that first, but the `binaryen` package is a dynamic version, and so, it's doesn't work.

Reply:
Okay, but in this case we don't need the package in the builder image

Reply:
Hmm, we need to simplify / consolidate the builder vs. base images I guess.